docs/source/install/ansible.rst

```diff
@@ -18,10 +18,10 @@ Supported Platforms
 
 Our Ansible playbooks support the same platforms as manual installation, i.e.:
 
-* Ubuntu Xenial (16.04)
```

Comment:
I don't think you want to remove Xenial from here? Should just be Trusty removed (which it looks like we did a while back)

Reply:
Sorry about that - realised I'd missed updating docs when I had added CentOS 8. And then when removing Trusty in playbooks thought needed a doc update. But seems the doc update on ansible.rst is ahead of the actual playbooks.

Technically the playbooks don't I believe support Bionic yet. But I've not removed that unless you want it removed...